_posts/2021-08-26-faas-storage-cassandra-astra.md

```diff
@@ -89,23 +91,23 @@ You can switch keyspace with the `use KEYSPACE;` command such as `use cycling;`
 ```sql
 // Insert a cyclist record record
 INSERT INTO cycling.cyclist_name (
-  id
-  lastname
+  id,
```

Comment:
This would have been copied and pasted, so why did it work for me, but not for you?

Reply:
All I can offer is that perhaps some commas were lost in the final editing of the post. CQL syntax requires the column list to be comma-separated (https://docs.datastax.com/en/cql-oss/3.3/cql/cql_reference/cqlInsert.html), indeed if I paste the statement as it is (both on standard Cassandra and Astra CQL Console) I get the syntax error
`SyntaxException: line 3:2 mismatched input 'lastname' expecting ')' (...INTO cycling.cyclist_name (  id  [lastname]...)`

Same for the fact that the values list has to be in the same order as the column list provided earlier: if the orderings don't correspond, CQL complains with something like
`InvalidRequest: Error from server: code=2200 [Invalid query] message="Type error: cannot assign result of function system.uuid (type uuid) to firstname (type text)"`

(I just tried with a regular Cassandra cluster as well as Astra DB to make sure both reproduce the same syntax errors consistently, and they do.)

Reply:
Thanks. Could this be because I wrapped the text after pasting it? Does it work without commas if all on one line?

Reply:
I tried that as well and, to my puzzlement, it does not seem to work even in that case.